packages/utils/src/common/comparators/__tests__/comparators.test.ts

```diff
@@ -0,0 +1,62 @@
+import { SortDirection, basicComparator, localizedStringComparator } from '@sequelize/utils';
+import { expect } from 'chai';
+
+describe('localizedStringComparator', () => {
```

Comment:
For a future PR we might want to look into a few additional tests for the other options. This PR is already large enough as it is